geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSInterIntegrationTest.java

```diff
@@ -41,6 +40,9 @@
   private JedisCluster jedis;
   private static final int REDIS_CLIENT_TIMEOUT =
       Math.toIntExact(GeodeAwaitility.getTimeout().toMillis());
+  private static final String REDIS_USER_SET1 = "{user1}set1";
+  private static final String REDIS_USER_SET2 = "{user1}set2";
+  private static final String REDIS_USER_SET3 = "{user1}set3";
```

Comment:
The naming of these constants is potentially a little misleading. The use of a tag at the beginning of the key is not related to users at all, but rather provides a way for Redis (or geode-for-redis) to guarantee that keys will be mapped to the same slot in the database ([see the documentation for the CLUSTER KEYSLOT command](https://redis.io/commands/cluster-keyslot)). These keys could just as well be "{tag}set1", "{tag}set2" etc. so having the constants be named "REDIS_USER_*" could cause confusion.

I think it might be best to change the constant names to just "SET1", "SET2" etc. and change the tag to something more generic so that there's no confusion in future.

Reply:
Thanks for the explanation. Changed the constants.